warehouse-management-service/pkg/database/postgres/init_test.go

```diff
@@ -14,7 +14,8 @@ const (
 )
 
 var warehouseService *WarehouseService
-var shelfService *ShelfBlockService
+var shelfBlockService *ShelfBlockService
```

Comment:
`shelfBlockService` is confusing when used with `shelfService`. Consider renaming to something else like `block` or `shelves`? I lack context on the entire domain, so will leave the name to your judgement.

Reply:
A `shelfBlock` can be thought of as a rack that contains multiple shelves. I had considered naming it `block` however, it did not feel descriptive enough